net.sourceforge.btanks.json

```diff
@@ -0,0 +1,91 @@
+{
+    "app-id": "net.sourceforge.btanks",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "finish-args": [
+        "--device=dri",
+        "--persist=.btanks",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/man",
+        "*.la", "*.a"
+    ],
+    "command": "btanks",
+    "rename-appdata-file": "btanks.appdata.xml",
+    "rename-desktop-file": "btanks.desktop",
+    "rename-icon": "btanks",
+    "modules": [
+        "shared-modules/glu/glu-9.0.0.json",
+        "shared-modules/SDL/SDL-1.2.15.json",
+        "shared-modules/smpeg/smpeg-0.4.5.json",
+        "shared-modules/SDL/SDL_image-1.2.12.json",
+        "shared-modules/lua5.3/lua-5.3.2.json",
+        {
+            "name": "scons",
+            "no-autogen": true,
```

Comment:
Yeah, that could work too, but that feature did not exist when I wrote this manifest :-)
Let's see if the rest of the build completes -- if it does perhaps we can just fix it later once we have a repo.